sdk/textanalytics/azure-ai-textanalytics/samples/sample_health.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_health.py
+
+DESCRIPTION:
+    This sample demonstrates how to detect healthcare entities in a batch of documents.
+    Each entity found in the document will have a link associated with it from a
+    data source.  Relations between entities will also be included in the response.
+
+USAGE:
+    python sample_health.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_TEXT_ANALYTICS_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_TEXT_ANALYTICS_KEY - your Text Analytics subscription key
+"""
+
+
+import os
+
+
+class HealthSample(object):
+
+    def health(self):
+        # [START health]
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.textanalytics import TextAnalyticsClient
+
+        endpoint = os.environ["AZURE_TEXT_ANALYTICS_ENDPOINT"]
+        key = os.environ["AZURE_TEXT_ANALYTICS_KEY"]
+
+        text_analytics_client = TextAnalyticsClient(
+            endpoint=endpoint, 
+            credential=AzureKeyCredential(key),
+            api_version="v3.2-preview.1")
+
+        documents = [
+            "RECORD #333582770390100 | MH | 85986313 | | 054351 | 2/14/2001 12:00:00 AM | \
+            CORONARY ARTERY DISEASE | Signed | DIS | Admission Date: 5/22/2001 \
+            Report Status: Signed Discharge Date: 4/24/2001 ADMISSION DIAGNOSIS: \
+            CORONARY ARTERY DISEASE. HISTORY OF PRESENT ILLNESS: \
+            The patient is a 54-year-old gentleman with a history of progressive angina over the past several months. \
+            The patient had a cardiac catheterization in July of this year revealing total occlusion of the RCA and \
+            50% left main disease , with a strong family history of coronary artery disease with a brother dying at \
+            the age of 52 from a myocardial infarction and another brother who is status post coronary artery bypass grafting. \
+            The patient had a stress echocardiogram done on July , 2001 , which showed no wall motion abnormalities ,\
+            but this was a difficult study due to body habitus. The patient went for six minutes with minimal ST depressions \
+            in the anterior lateral leads , thought due to fatigue and wrist pain , his anginal equivalent. Due to the patient's \
+            increased symptoms and family history and history left main disease with total occasional of his RCA was referred \
+            for revascularization with open heart surgery."
+        ]
+
+        poller = text_analytics_client.begin_health(documents)
+        result = poller.result()
+        docs = [doc for doc in result if not doc.is_error]
+
+        for idx, doc in enumerate(docs):
```

Comment:
This is a good sample to show the general information that comes back form healthcare. What is missing is a user scenario behind it.
who will use this API and how they will use it. the closes example I think we have in Python is for OM => https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze_sentiment_with_opinion_mining.py

See how there is a story (user scenario) and how there is manipulation on the data instead of just printing it

Reply:
Thanks for sharing the opinion mining sample!  I think that makes a lot of sense to have for this.  I think I have some idea at a high level of what this means for healthcare, but the specifics may take some time to develop. 

In the interest of time for the first iteration, is this something that is critically needed or can it be a part of the larger design effort that we will do later?

Reply:
Good point. I chatted with @iscai-msft and for beta.1 it is ok to have this sample and improve in the larger design effort

Reply:
@abahn we can still leave feedback here if there's anything we want to say, but this feedback doesn't have to be addressed for preview 1, just create an issue if you want to address it later